Client/Info.plist

```diff
@@ -3,15 +3,15 @@
 <plist version="1.0">
 <dict>
 	<key>AdjustAppToken</key>
-	<string>$(ADJUST_APP_TOKEN)</string>
+	<string></string>
 	<key>AdjustEnvironment</key>
-	<string>$(ADJUST_ENVIRONMENT)</string>
+	<string></string>
 	<key>AppIdentifierPrefix</key>
 	<string>$(APP_IDENTIFIER_PREFIX)</string>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
 	<key>CFBundleDisplayName</key>
-	<string>$(MOZ_BUNDLE_DISPLAY_NAME)</string>
+	<string>$(DISPLAY_NAME)</string>
```

Comment:
Moved the old display name property from the now-removed .xcconfig file to a user defined variable for this target.